tests/gtest/aviftest_helpers.h

```diff
@@ -13,6 +13,17 @@ namespace libavif
 namespace testutil
 {
 
+enum avifChannel
+{
+    R = 0,
+    G = 1,
+    B = 2,
+    A = 3
+};
+
+// Maps from avifChannel to sample position in avifRGBFormat.
+uint32_t avifChannelOffset(avifRGBFormat format, avifChannel channel);
```

Comment:
Please name this function `GetRGBChannelOffset`. The `avif` prefix is not necessary because this is a C++ function in the `libavif` namespace.

If I understand it correctly, the code in this pull request always get the R, G, B, A channel offsets at the same time. So we should consider a function that returns the four channel offsets in one shot, for example,

```
struct RGBChannelOffsets {
  uint8_t r;
  uint8_t g;
  uint8_t b;
  uint8_t a;
};

RGBChannelOffsets GetRGBChannelOffsets(avifRGBFormat format);
```

And then we pass `uint32_t channelOffset` instead of `avifChannel channel` to functions like `modifyImageChannel`, `setImageChannel`, and `fillImageChannel`. And we won't need the `avifChannel` enum type.